src/constants/defaultLookup.js

```diff
@@ -0,0 +1,70 @@
+import {
+  FILTERS,
+  REFERENCES_VALUES_MAP,
+  searchableIndexesValues,
+} from '@folio/stripes-authority-components';
+
+export const DEFAULT_FILTERS = {
+  [FILTERS.REFERENCES]: [
+    REFERENCES_VALUES_MAP.excludeSeeFrom,
+    REFERENCES_VALUES_MAP.excludeSeeFromAlso,
+  ],
+};
+
+const LCNAF = {
+  [FILTERS.AUTHORITY_SOURCE]: ['af045f2f-e851-4613-984c-4bc13430454a'],
+};
+
+const CORPORATE_CONFERENCE_NAME_SET = {
+  qindex: searchableIndexesValues.CORPORATE_CONFERENCE_NAME,
+  filters: LCNAF,
+};
+
+const UNIFORM_TITLE_SET = {
+  qindex: searchableIndexesValues.UNIFORM_TITLE,
+  filters: LCNAF,
+};
+
+const PERSONAL_NAME_SET = {
+  qindex: searchableIndexesValues.PERSONAL_NAME,
+  filters: LCNAF,
+};
+
+export const DEFAULT_LOOKUP_OPTIONS = {
```

Comment:
I think this is a good approach in general. I'd just like to move this logic to `ui-quick-marc` because the plugin should be as separated as possible from the linking logic so it can be used for other purposes if need be
I know we have defined the link buttons as part of the component, and it can be considered a tech dept. I just don't want to increase it further by coupling the plugin with linking too much

Reply:
Do you mean moving and using constants from `@folio/quick-marc/src/common/constants`?
`import {
   DEFAULT_FILTERS, 
   DEFAULT_LOOKUP_OPTIONS
} from '@folio/quick-marc/src/common/constants';`